packages/framework/ai-collab/src/aiCollabApi.ts

```diff
@@ -200,6 +207,11 @@ export interface AiCollabErrorResponse {
 	 * {@inheritDoc TokenUsage}
 	 */
 	readonly tokensUsed: TokenUsage;
+	/**
+	 * A list of diffs that represent the changes made by the AI collaboration.
+	 * @remarks This is intended to be used to update your applications UI view with the changes made by the AI collaboration.
+	 */
+	readonly uiDiffs: Diff[];
```

Comment:
Can this be a readonly array?
```suggestion
	readonly uiDiffs: readonly Diff[];
```

Reply:
Oh? the property is already defined as readonly. Why there is extra readonly data type?

Reply:
Updated

Reply:
> Oh? the property is already defined as readonly. Why there is extra readonly data type?

The first `readonly` makes it so the object's property cannot be modified. The second `readonly` changes the type of the array to prevent modification of the array's contents.